bindings-go/Makefile

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+GO_CMD=go
+GO_BUILD=$(GO_CMD) build
+CMD_PATH=./gen/main.go
+DIST=dist
+BINARY_NAME=bindings
+
+PKG="bindings"
+MONOREPO_BASE=`git rev-parse --show-toplevel | xargs`
+CONTRACTS_DIR="${MONOREPO_BASE}/contracts"
+GETH_DIR := "${MONOREPO_BASE}/services/el_clients/go-ethereum"
+
+.PHONY: all # Run all necessary steps and generate bindings
+all: bindings
+
+.PHONY: abigen # Install latest abigen command binary
+abigen:
+	cd "${GETH_DIR}" && \
+		go run build/ci.go install ./cmd/abigen
```

Comment:
where will this be installed?

Reply:
initially I wanted to do this by a download, but I run into issues, so I left this same as earlier in the /ops - just a compilation from our go-ethereum fork; it's not installed, just used later as arg in `-abigen-bin "${GETH_DIR}/build/bin/abigen"` of the `bindings` action